include/aca_zeta_programming.h

```diff
@@ -27,11 +27,28 @@ static atomic_uint current_available_group_id(1);
 
 namespace aca_zeta_programming
 {
+struct fwd_info {
```

Comment:
this looks like a c structure but it is using c++ feature operator overload. Please consider switching to c++ class.